templates/unicorn.rb.j2

```diff
@@ -1,102 +1,25 @@
-# Sample verbose configuration file for Unicorn (not Rack)
-#
-# This configuration file documents many features of Unicorn
-# that may not be needed for some applications. See
-# http://unicorn.bogomips.org/examples/unicorn.conf.minimal.rb
-# for a much simpler configuration file.
-#
-# See http://unicorn.bogomips.org/Unicorn/Configurator.html for complete
-# documentation.
+# {{ansible_managed}}
 
-# Note: If you change this file in a Merge Request, please also create a
-# Merge Request on https://gitlab.com/gitlab-org/omnibus-gitlab/merge_requests
-
-# Relative URL support
-# WARNING: We recommend using an FQDN to host GitLab in a root path instead
-# of using a relative URL.
-# Documentation: http://doc.gitlab.com/ce/install/relative_url.html
-# Uncomment and customize the following line to run in a non-root path
-#
-# ENV['RAILS_RELATIVE_URL_ROOT'] = "/gitlab"
-
-# Read about unicorn workers here:
-# http://doc.gitlab.com/ee/install/requirements.html#unicorn-workers
-#
 worker_processes {{ gitlab_unicorn_worker_processes }}
```

Comment:
document or at least mention all the unicorn vars or add defaults / default filters for lists.